spec/spec_helper.rb

```diff
@@ -6,6 +6,7 @@
 require "minitest/spec"
 require "minitest/hooks/default"
 require "minitest/reporters"
+require "byebug"
```

Comment:
Why do we need this?

Reply:
Every time when debugging I forget that we don't require byebug for tests and need to add it. We have it in the Gemfile, we are just not requiring it in the spec helper.